### ISSUE TITLE
temp file in 2 places

### ISSUE THREAD

**Issue by JohnBilberry** [cclamb/thermocouple#1]
The temp file is written in two places, /var/log/temp and in /tmp/temp.

Not sure if this is intentional?

**Comment by cclamb** [comment]
Could you double check that this is actually writing the temp file you see in /var/log? I don't think it is. I say this because the filenames are defined at the top of tc_main.c, and they both write to /tmp. Perhaps the /var/log/temp file was created by something else? what are the contents?

**Comment by JohnBilberry** [comment]
When I run tcsimd on my local machine it creates a file /tmp/temp, but not in var/log.
When I run tcsimd on my image it creates a file in /tmp/temp and in /var/log/temp.
I can kill tcsmid and delete those files, when I start tcsimd again, they reappear.
If I delete them while tcsimd is running, they reappear.
Is it possible somehow on my image the tmp and var/log are mirroring one another?
Because they have the same files in them. Is this normal? 
Thus far I am still using the image we built in class.

**Comment by cclamb** [comment]
Yeah, one is likely soft/hard linked to the other in your image. Run 'ls -alh /var and 'ls -alh /' and post the results.

**Comment by JohnBilberry** [comment]
$ ls -alh /var
total 5
drwxr-xr-x    5 root     root        1.0K Mar 26  2018 .
drwxr-xr-x   19 root     root        1.0K Jan  1 00:08 ..
lrwxrwxrwx    1 root     root           6 Mar  4  2018 cache -> ../tmp
drwxr-xr-x    2 root     root        1.0K Mar 26  2018 empty
drwxr-xr-x    2 root     root        1.0K Mar  4  2018 lib
lrwxrwxrwx    1 root     root           6 Mar  4  2018 lock -> ../tmp
lrwxrwxrwx    1 root     root           6 Mar  4  2018 log -> ../tmp
lrwxrwxrwx    1 root     root           6 Mar  4  2018 run -> ../run
lrwxrwxrwx    1 root     root           6 Mar  4  2018 spool -> ../tmp
lrwxrwxrwx    1 root     root           6 Mar  4  2018 tmp -> ../tmp
drwxr-xr-x    2 www-data www-data    1.0K Mar 26  2018 www
$ ls -alh /
total 26
drwxr-xr-x   19 root     root        1.0K Jan  1 00:08 .
drwxr-xr-x   19 root     root        1.0K Jan  1 00:08 ..
drwxr-xr-x    2 root     root        2.0K Apr 26  2018 bin
drwxr-xr-x    6 root     root        2.5K Jan  1 00:00 dev
drwxr-xr-x    7 root     root        1.0K Jan  1 00:00 etc
drwxr-sr-x    2 john     john        1.0K Jan  1 00:09 john
drwxr-xr-x    3 root     root        1.0K Apr 26  2018 lib
lrwxrwxrwx    1 root     root           3 Mar 26  2018 lib32 -> lib
lrwxrwxrwx    1 root     root          11 Mar 26  2018 linuxrc -> bin/busybox
drwx------    2 root     root       12.0K Apr 26  2018 lost+found
drwxr-xr-x    2 root     root        1.0K Mar  4  2018 media
drwxr-xr-x    2 root     root        1.0K Mar  4  2018 mnt
drwxr-xr-x    2 root     root        1.0K Mar  4  2018 opt
dr-xr-xr-x   39 root     root           0 Jan  1 00:00 proc
drwx------    2 root     root        1.0K Jan  1 00:01 root
drwxr-xr-x    3 root     root         180 Jan  1 00:00 run
drwxr-xr-x    2 root     root        1.0K Apr 26  2018 sbin
dr-xr-xr-x   12 root     root           0 Jan  1 00:00 sys
drwxrwxrwt    2 root     root         140 Jan  1 00:32 tmp
drwxr-xr-x    7 root     root        1.0K Apr  4  2018 usr
drwxr-xr-x    5 root     root        1.0K Mar 26  2018 var

**Comment by JohnBilberry** [comment]
I mean it works regardless.
